geode-lucene/build.gradle

```diff
@@ -19,11 +19,12 @@ apply from: "${rootDir}/${scriptDir}/standard-subproject-configuration.gradle"
 
 apply from: "${project.projectDir}/../gradle/publish-java.gradle"
 
+apply plugin: 'geode.jboss-modules-plugin'
 
 dependencies {
   api(platform(project(':boms:geode-all-bom')))
 
-  api(project(':geode-core'))
+  implementation(project(':geode-core'))
```

Comment:
`geode-lucene` creates `LuceneIndexDestroyedException`, which extends `GemFireException`.  This requires `geode-core` to remain an `api` dependency.

Reply:
Thank you. Will update